.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,7 +18,7 @@ Please check the type of change your PR introduces:
 
 <!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
 
-Issue Number: N/A
+Resolves: N/A
```

Comment:
does it work with ":" in between resolves and issue number?

Reply:
not sure, I've updated to be sure that we don't do twice the same PR...